build_ts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "warp-workflows",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I just published warp-workflows at version `1.0.0` so bumping this up by a version so I can go ahead and publish again.